### PR TITLE
fix initialization of link stats

### PR DESCRIPTION
**Pull request by atiselsts**

### CODE REVIEW

File: os/net/link-stats.c [hunk 1]
```diff
@@ -147,6 +147,7 @@ link_stats_packet_sent(const linkaddr_t *lladdr, int status, int numtx)
     /* Add the neighbor */
     stats = nbr_table_add_lladdr(link_stats, lladdr, NBR_TABLE_REASON_LINK_STATS, NULL);
     if(stats != NULL) {
+      memset(stats, 0, sizeof(*stats));
```
Comment (simonduq) [inline os/net/link-stats.c]: the two `memset` should not be necessary, as this is already done by `nbr_table`. Or am I missing something?

Comment (atiselsts) [reply os/net/link-stats.c]: True, the nbr table code does memset already.

**Comment by atiselsts** [comment]
Perhaps don't merge this yet, I have more link stats changes in the Amber project that I'll ask to be released.

**Comment by simonduq** [comment]
Is this ready?

**Comment by atiselsts** [comment]
Yes, I believe so.

**Pull request merged**